dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/program/hibernate/ProgramNotificationTemplate.hbm.xml

```diff
@@ -36,6 +36,8 @@
 
     <property name="notifyUsersInHierarchyOnly" column="notifyusersinhierarchyonly" />
 
+    <property name="sendRepeatable" column="sendrepeatable" />
```

Comment:
not-null="true" if its actually a non nullale column

Reply:
resolved